PLUGINS.md

```diff
@@ -0,0 +1,35 @@
+# Plugins
+
+The iXBRL Viewer provides a simple plugin framework, to enable projects to
+build a viewers with additional functionality.
```

Comment:
Weird grammer

Reply:
Fixed in 40a56e7b5353864393a65eeb7bb8ea5a4fecc98f